htdocs/fichinter/class/fichinter.class.php

```diff
@@ -1526,7 +1526,7 @@ function update_total()
 
 		$sql = "SELECT SUM(duree) as total_duration, min(date) as dateo, max(date) as datee ";
 		$sql.= " FROM ".MAIN_DB_PREFIX."fichinterdet";
-		$sql.= " WHERE fk_fichinter=".$this->fk_fichinter;
+		$sql.= " WHERE fk_fichinter=".$this->id;
```

Comment:
fk_fichinter is an id of fichinter table and this->id is an id of line table. Why mixing apple and pie ?

Reply:
$this is the object fichinter, who don't have fk_fichinter key
we confuse the fact to use fk_fichinter field of the table fichinterdet and the id of the object Fichinter
It's just a bug...